packages/account/src/Components/self-exclusion/__tests__/self-exclusion.spec.tsx

```diff
@@ -2,30 +2,68 @@ import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import SelfExclusion from '../self-exclusion';
+import { FormikValues } from 'formik';
 
 const portal_root = document.createElement('div');
 document.body.appendChild(portal_root);
 
 jest.mock('Stores/connect.js', () => ({
     __esModule: true,
     default: 'mockedDefaultExport',
-    connect: () => Component => Component,
+    connect: () => (Component: React.Component) => Component,
 }));
 
 jest.mock('@deriv/shared', () => ({
     ...jest.requireActual('@deriv/shared'),
     useIsMounted: jest.fn().mockImplementation(() => () => true),
 }));
 
-jest.mock('../self-exclusion-modal', () => () => <div>SelfExclusionModal</div>);
+jest.mock('../self-exclusion-modal', () => {
+    const MockSelfExclusionModal = () => <div>SelfExclusionModal</div>;
+    return MockSelfExclusionModal;
```

Comment:
If i didn't declare a const name to the function, it will return error to me stating that the function needs a name. The only way to solve the warning is to provide a const variable to it and return the value